File: Ruby/1036 - Fórmula de bhaskara.rb
info = gets.split(' ')

a = info[0].to_f
b = info[1].to_f
c = info[2].to_f

delta = (b**2) - 4*a*c

if delta > 0 and a != 0 then
	r1 = (-b + (delta ** 0.5)) / (2*a)
	r2 = (-b - (delta ** 0.5)) / (2*a)

	puts "R1 = #{format("%.5f", r1)}"
	puts "R2 = #{format("%.5f", r2)}"
else 
	puts "Impossivel calcular"
end

File: Go/1036 - Fórmula de Bhaskara.go
package main

import (
	"fmt"
	"math"
)

func main(){

	var a, b, c, r1, r2, delta float64

	fmt.Scan(&a, &b, &c)

	delta = (b*b) - 4*a*c 

	if delta > 0 && a != 0 {
		
		r1 = (-b + math.Sqrt(delta)) / (2*a)
		r2 = (-b - math.Sqrt(delta)) / (2*a)

		fmt.Printf("R1 = %.5f\n", r1)
		fmt.Printf("R2 = %.5f\n", r2)

	} else {
		fmt.Printf("Impossivel calcular\n")
	}
}